app/res/layout/progress_dialog_cancel_button.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Button xmlns:android="http://schemas.android.com/apk/res/android"
-        android:layout_width="wrap_content"
-        android:layout_height="@dimen/min_button_height"
-        android:layout_below="@+id/main_content_holder"
-        android:layout_alignParentRight="true"
-        android:layout_marginBottom="@dimen/standard_spacer"
-        android:layout_marginRight="@dimen/standard_spacer_large"
-        android:padding="@dimen/standard_spacer"
-        android:background="@android:color/transparent"
-        android:text="STOP"
-        android:textSize="@dimen/font_size_dp_medium"
-        android:visibility="gone"
+    android:layout_width="wrap_content"
+    android:layout_height="@dimen/min_button_height"
+    android:layout_below="@+id/main_content_holder"
+    android:layout_alignParentEnd="true"
+    android:layout_alignParentRight="true"
     android:layout_marginEnd="@dimen/standard_spacer_large"
-    android:layout_alignParentEnd="true">
-</Button>
\ No newline at end of file
+    android:layout_marginRight="@dimen/standard_spacer_large"
+    android:layout_marginBottom="@dimen/standard_spacer"
+    android:background="@android:color/transparent"
+    android:padding="@dimen/standard_spacer"
+    style="@style/CommCare.Button.Default"
+    android:text="STOP"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/res/layout/progress_dialog_cancel_button.xml#L14"></span>Hardcoded string "STOP", should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>